playbooks/roles/scorm/templates/sync-scorm-content-to-s3.j2

```diff
@@ -62,4 +62,4 @@ onerror() {
 
 trap onerror ERR SIGHUP SIGINT SIGTERM
 
-s3cmd sync -c {{ scorm_dirs.home.path }}/.s3cfg $noop {{ scorm_asset_local_storage_path }}/ "s3://{{ EDXAPP_AWS_STORAGE_BUCKET_NAME }}/scorms/" >>{{scorm_s3_sync_logfile}} 2>&1
+s3cmd sync -c {{ scorm_dirs.home.path }}/.s3cfg $noop {{ scorm_asset_local_storage_path }}/ "s3://{{ EDXAPP_AWS_STORAGE_BUCKET_NAME }}/scorms/" >>{{scorm_s3_sync_logfile}} 2>&1
```

Comment:
Niptuck: Same noise here as well. (A newline was removed, which is not necessary and clobbers the git log).

Reply:
removed.